Client/Frontend/Browser/OpenWithSettingsViewController.swift

```diff
@@ -0,0 +1,134 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+class OpenWithSettingsViewController: UITableViewController {
+
+    private lazy var mailSchemes: [[String:String]] = {
+        var plistSchemes: [[String:String]] = self.loadMailSchemes()
+        return plistSchemes
+    }()
+    private var mailSchemeEnableCache: [String:Bool] = [String:Bool]()
```

Comment:
Nit: Maybe just personal preference but this is named a bit weird. Maybe `enabledMailSchemes`?